harness/determined/exec/harness.py

```diff
@@ -149,6 +132,6 @@ def main(train_entrypoint: Optional[str]) -> int:
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("--train-entrypoint")
+    parser.add_argument("train-entrypoint")
```

Comment:
IIRC, i think this "--named_arg" is needed by argparse if you have a "-" in the argument name like "named-arg".

Reply:
Um, I don't think that's accurate

Reply:
well shut my mouth... I'll fix this.

Reply:
fixed, using `"train_entrypoint"` now, thank you, and sorry for my sass lol